Source/WebInspectorUI/UserInterface/Views/FontDetailsPanel.js

```diff
@@ -269,7 +269,7 @@ WI.FontDetailsPanel = class FontDetailsPanel extends WI.StyleDetailsPanel
             return new WI.DetailsSectionSimpleRow(WI.UIString("Style", "Style @ Font Details Sidebar Property", "Property title for `font-style`."));
             break;
         case "font-weight":
-            return new WI.DetailsSectionSimpleRow(WI.UIString("Weight", "Weight @ Font Details Sidebar Property", "Property title for `font-weight`."));
```

Comment:
The change in `FontDetailsPanel.js` looks fine. It's an update to the comment shown to translators (no change in the UI). But it's missing from the `Localizations/en.lproj/localizedStrings.js`. 

It can be automatically updated by running the script:

`Tools/Scripts/extract-localizable-js-strings --utf8 Source/WebInspectorUI/Localizations/en.lproj/localizedStrings.js Source/WebInspectorUI/UserInterface`

Reply:
I ran that script several times before realizing that it won't have any effect because this is the comment that is already in there. I'm deduplicating here after all. 😊

Reply:
Ah, didn't realize that. I was just looking for a corresponding diff in the patch. All good, then 👍